src/components/Login/login.scss

```diff
@@ -0,0 +1,104 @@
+
+.Login {
+    display: flex;
+    padding: 44px 100px;
+    justify-content: space-around;
+}
+  
+.Login__info {
+    display: none;
+
+    h2 {
+        line-height: 1.1;
+    }
+}
+  
+.Login__info__text::before {
+    content: ' ';
+    position: absolute;
+    background: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAeCAYAAAA7MK6iAAAABmJLR0QA/wD/AP+gvaeTAAACcklEQVRIx+1VS2sUQRBu3+Jzs109IxGMb/QiiBdRBC+CHlTwoBdBgqIn8RrwEhTEi+BFFPEXrCBI2N3pmZVBELwsRAnz2I0a9aABxQcqGk0yVu/WPBLNRmPnIGxBs8z37dRXVV1Vw1jb2vY/WWjxw77kF4fctYuzeLWvc4nCAwkHZ0FU7EfHo3giX4pjMR5FbE4g+W2F4/mgVbTu8K3KKTl/NVAxzJjD517CI9+G69pEPWtFHp3WyflX38nvTETL/Ahi48Q9GCxtXKRF1HXZ/NCCSpxRaPPjSUCO2I7Y5wZnw/Mn0jT03asU12LRQIpLiWhRrAolvCTuU1g2tmkTxQY6mdydhFJUYPMUrjoag3hI3JjqdI33Kvag0xEqo/fU6ViZVgFupVXgPRrLa64LLHhD5X07WDI2pB3Me1JRKKhR0iN6F5aj2AA5/h6UYW/M1aQ4EM8xnqpaGlpEo142Fx32JRnZcGaqOa7d46tbdeR5vI+iX8x1/YkwlvdK0kwWv5pk6nYCYs+I+4JnR2tHEj5SAwxNJ47rsDvJ1OJW3MFegS1E7D5x4/i/o9OPgw2n0q0ytXhoG7vxP9/Iedjv5nKZ4G8my8OCC387i2P08ot6xVw/gcdgEB8m/l1Qgc1pQOJcZnncUT0wk0Xwi7jnimX4/IjwUdW16Tv5fYj9aGQqof+xNJfOqGPRwels2RuzillkMjpbk3xLaMOhxq+E98QNe07Hmn8al4nitNyb5wbxrydxeO/GLi2zOklczapbrbIFzfJCKROMqswJptNIXG2guprRGFf3qAJpfv7EZTYbpu7tdw2jvr++LTaxtrWthf0Eq7n8fF4wMD4AAAAASUVORK5CYII') no-repeat;
+    top: 0;
+    left: 0;
+    width: 20px;
+    height: 20px;
+    background-size: 100%;
+}
+  
+.Login__form__header {
+    text-align: center;
+    margin: 0 auto;
+    line-height: 62px;
+    font: {
+        style: normal;
+        weight: normal;
+        size: 50px;
+    }
+}
+  
+.Login__form {
+    h1{
+        text-align: center;
+    }
+    width: 100%;
+    max-width: 400px;
+
+    .form {
+        margin: 20px 0;
+        padding: 33px;
+        border: 1px solid rgba(0, 0, 0, 0.2);
+        box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+        line-height: 1.3;
+        font-size: 1em;
+    }
+
+
+    .form-field:last-of-type {
+        text-align: center;
+    }
+}
+
+
+  
+/* not small devices  */
+@media (min-width: 800px) {
+    .Login__info {
+        display: block;
+        align-self: center;
+        padding-right: 1rem;
+    }
+
+    .Login__info__header {
+        font-size: 2em;
+    }
+
+    .Login__info__text {
+        position: relative;
+        padding-left: 30px;
+    }
+
+    .Login__form {
+        padding-left: 1rem;
+    }
+}
+
+.form__footer, .form__footer span:nth-child(1) {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+ }
+ .form__footer > span:nth-child(1) {
+    flex-direction: row !important;
+   margin: 15px 0
+ }
+ span > a {
+    margin: 5px
+  }
+.center{
+    margin: 0 auto;
+    text-align: center;
+}
+.input-validation-error {
+    font-size: 0.95em;
+    color:red !important;
```

Comment:
@jhayX You didn't remember to replace the `red` value with the appropriate color.